test/suite/test_chunkcache02.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```

Comment:
Should we rename this to test_chunkcache01 ?

Reply:
@korteland added a Python test with that name in https://github.com/wiredtiger/wiredtiger/pull/9529